app/src/main/res/layout/fragment_control_advanced.xml

```diff
@@ -619,7 +631,8 @@
                     android:text="@string/sqr2"
                     android:textSize="18sp"
                     android:textStyle="bold"
-                    android:layout_marginStart="12dp"/>
+                    android:layout_marginStart="12dp"
```

Comment:
Yes. I will do it along with other changes. Actually, I will be replacing lot of the repetitive stuff by creating a custom component.